src/components/seekbar/seekbar.tsx

```diff
@@ -138,6 +138,7 @@ class SeekBar extends Component<any, any> {
   componentWillUnmount(): void {
     document.removeEventListener('mouseup', this.onPlayerMouseUp);
     document.removeEventListener('mousemove', this.onPlayerMouseMove);
+    this.props.eventManager.unlisten(document, 'bottomBarClientRectChanged', this.handleUpdateSeekBarClientRect);
```

Comment:
actually, and you can double check, when you use withEventManager, it [should remove the event listeners by itself on unmount](https://github.com/kaltura/playkit-js-ui/blob/fa0edb505f9230b953011a916ce090e6f0cc09f6/src/event/with-event-manager.tsx#L22), without you having to do it in your code

also, when you export the event name to EventType, please use EventType.[new event name] here instead of writing the string. it will just make refactor easier if the string can be changed in one place and affect every place that uses the event.

also that way we can reuse this event in plugins if we need to, by just importing EventType in that plugin.